modules/ROOT/pages/authentication-authorization/privileges-and-roles-immutable.adoc

```diff
@@ -0,0 +1,150 @@
+[role=label--new-5.24]
```

Comment:
```suggestion
[role=label--new-5.26]
```

Reply:
Yes, I've already addressed those in my local branch.